acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,44 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
```

Comment:
Using 'require' appears to work for the Unix's, but not for the Windows boxes as the puppet ruby doesn't have facter.rb in its $LOAD_PATH.